ftw/crawler/configuration.py

```diff
@@ -75,6 +76,11 @@ def __init__(self, url, attributes=None, sleeptime=0.1):
     def bind(self, config):
         self.config = config
 
+    @property
+    def base_url(self):
+        parsed_url = urlparse(self.url)
+        return '{url.scheme}://{url.netloc}'.format(url=parsed_url)
```

Comment:
Nice! I did not know you could access the argument's attributes with `format`.
Found this https://pyformat.info/ and discovered that format has even more features I did not know of.
Thank you!